package.json

```diff
@@ -181,6 +181,30 @@
   "pnpm": {
     "overrides": {
       "usocket": "_EXCLUDED_"
+    },
+    "peerDependencyRules": {
+      "allowedVersions": {
+        "esbuild-sass-plugin>esbuild": "^0.16.7",
+        "react-electron-web-view>react": "^18.0.0",
+        "react-electron-web-view>react-dom": "^18.0.0",
+        "react-loader>react": "^18.0.0",
+        "react-loader>react-dom": "^18.0.0",
+        "react-sortable-hoc>react": "^18.0.0",
+        "react-sortable-hoc>react-dom": "^18.0.0"
```

Comment:
All these dependencies are pretty much deprecated/unmaintained (don't support React 18) and should be switched out with alternatives.

Reply:
please add TODO comments in the code - once this PR is closed, we will lose track of such info

Reply:
`package.json` doesn't allow comments unfortunately, where would you like me to add this?

Reply:
shall we create issues so that this is not lost?

Reply:
Yeah, that was my plan, I will create an issue for it, maybe later in the day.